storybook/entry.js

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable prefer-destructuring */
```

Comment:
Would be nice linking to base source (mentioning version/license too)

Reply:
Also is there any way that we can pull original entry file and create an extended `entry.nuxt.js`? This way we can keep with future updates easier. Or at least highlighting sections we add/modify

Reply:
Pulling is not an option. But I'll highlight sections :)

Reply:
I see. Because of [this line](https://github.com/storybookjs/storybook/blob/next/app/vue/src/client/preview/index.ts#L117) we probably can't just import

Reply:
Also in [render.ts](https://github.com/storybookjs/storybook/blob/next/app/vue/src/client/preview/render.ts#L8)